recipes/conda-wrappers/meta.yaml

```diff
@@ -0,0 +1,28 @@
+{% set version = "1.0.0" %}
+
+package:
+  name: conda-wrappers
+  version: {{ version }}
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - python
+  run:
+    - python
+    - exec-wrappers 1*
```

Comment:
Not quite `1*` could mean `10` or similar. This could cause for some strange surprises later. Hence we have preferred the `.`.

If the `.` is not always there, we do something like `1|1.*`.